<launch>
    <env name = "GSCAM_CONFIG" value = "v4l2src device=/dev/video0 !
        video/x-raw-rgb,framerate-25/1 ! ffmpegcolorspace" />

    <node pkg="gscam" name="web_cam" type="gscam">
        <param name = "camera_name" value = "web_cam" />
    </node>

    <node pkg="assignment1" type="better_object_detection.py" name="ball_finder" output="screen" />

    <node pkg="rqt_image_view" type="rqt_image_view" name="rqt_image_view" /> 

    <node pkg="assignment1" type="arm_controller.py" name="arm_controller" output="screen" />

</launch>
